allensdk/brain_observatory/behavior/data_objects/running_speed/running_acquisition.py

```diff
@@ -117,7 +125,10 @@ def to_json(self) -> dict:
             )
         output_dict = dict()
         output_dict.update(self._stimulus_file.to_json())
-        output_dict.update(self._stimulus_timestamps.to_json())
+        if self._stimulus_timestamps is not None:
+            if self._stimulus_timestamps._sync_file is not None:
+                output_dict.update(
+                    self._stimulus_timestamps._sync_file.to_json())
```

Comment:
Are we leaving these `to_json`s sitting around since we aren't really implementing them yet as we discussed?

Reply:
Yes, it really shouldn't have been added in the first place. But if we want to start implementing `to_json`, we'll have to revisit this.